File: codegen/scala-gen/src/test/resources/tests/view-service/view-in-name-new-style/generated-unmanaged/org/example/view/UserByNameViewImpl.scala
package org.example.view

import kalix.scalasdk.view.View.UpdateEffect
import kalix.scalasdk.view.ViewContext

// This class was initially generated based on the .proto definition by Kalix tooling.
//
// As long as this file exists it will not be overwritten: you can maintain it yourself,
// or delete it so it is regenerated as needed.

class UserByNameViewImpl(context: ViewContext) extends AbstractUserByNameView {

  override def emptyState: UserState =
    throw new UnsupportedOperationException("Not implemented yet, replace with your empty view state")

  override def updateCustomer(
      state: UserState,
      userState: UserState): UpdateEffect[UserState] =
    throw new UnsupportedOperationException("Update handler for 'UpdateCustomer' not implemented yet")

}

File: codegen/java-gen/src/test/resources/tests/view-service/view-in-name-new-style/generated-unmanaged/org/example/view/UserByNameViewImpl.java
package org.example.view;

import kalix.javasdk.view.View;
import kalix.javasdk.view.ViewContext;

// This class was initially generated based on the .proto definition by Kalix tooling.
// This is the implementation for the View Service described in your example-views.proto file.
//
// As long as this file exists it will not be overwritten: you can maintain it yourself,
// or delete it so it is regenerated as needed.

public class UserByNameViewImpl extends AbstractUserByNameView {

  public UserByNameViewImpl(ViewContext context) {}

  @Override
  public UserViewModel.UserState emptyState() {
    throw new UnsupportedOperationException("Not implemented yet, replace with your empty view state");
  }

  @Override
  public View.UpdateEffect<UserViewModel.UserState> updateCustomer(
      UserViewModel.UserState state,
      UserViewModel.UserState userState) {
    throw new UnsupportedOperationException("Update handler for 'UpdateCustomer' not implemented yet");
  }

}

